lib/tfpdf/wp-tfpdf.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * To avoid modifying tFPDF directly we introduce WP_tFPDF, 
+ * A thin wrapper around the parent class to utilize WP_Filesystem 
+ * 
+ * Parent's coding standards are preserved deliberately.
+ */
+// phpcs:disable WordPress.WP.AlternativeFunctions.file_system_read_fwrite, WordPress.WP.AlternativeFunctions.file_system_read_fread, WordPress.WP.AlternativeFunctions.file_system_read_fopen, WordPress.WP.AlternativeFunctions.file_system_read_fclose, WordPress.VIP.FileSystemWritesDisallow
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+require_once ABSPATH . '/wp-admin/includes/file.php';
+
+class WP_tFPDF extends tFPDF {
+	function __construct( $orientation = 'P', $unit = 'mm', $size = 'A4' ) {
+		parent::__construct( $orientation, $unit, $size );
+		$this->init_wp_filesystem();
+	}
+	
+	function init_wp_filesystem() {
```

Comment:
Should this be `private`?

Reply:
No particular reason for it to be or not to be private, since it operates on a global `$wp_filesystem`

Reply:
I'm thinking that this method shouldn't be used by 3rd-party code, and should only be called from the constructor. So making it private is probably better from a maintenance standpoint. That way, if we want to change it, we don't have to worry about other code that might be using it for some reason 🙂